#
# Image Space Potential field parameters
#

# Arbitrary measurement value to assign as reward for targets.
target_reward: 50.0

hard_constraint_transform:
  translation: 0.0
  alpha: 25.0
  beta: 1.0
  range_min: 0.0
  range_max: 5.0

soft_constraint_transform:
  translation: 0.0
  alpha: 1.0
  beta: 1.0
  range_min: 5.0
  range_max: 50.0

